src/ape/managers/chain.py

```diff
@@ -370,8 +370,12 @@ def pending_timestamp(self) -> int:
         return self.provider.get_block("pending").timestamp
 
     @pending_timestamp.setter
-    def pending_timestamp(self, new_value: int):
-        self.provider.set_timestamp(new_value)
+    def pending_timestamp(self, new_value: str):
+        """
+        Converts date string to timestamp
+        """
+        from ape import convert
+        convert(value=new_value, type=int)
```

Comment:
You deleted the line that was doing the work lol
```suggestion
        convert(value=new_value, type=int)
        self.provider.set_timestamp(new_value)
```

Reply:
Does this change new_value to a timestamp, or does this return a value?
I think it should be `self.provider.set_timestamp(convert(value=new_value, type=int))`